Typescript warnings because of `NodeJS.EventEmitter`

I recently upgraded `@types/node` to `16.11.11` and started to notice the following two errors.

<details>
  <summary>Expand to see logs</summary>
  <pre>
gitpod /workspace/webstone/packages/cli $ pnpm prepublishOnly

> @webstone/cli@0.7.0 prepublishOnly /workspace/webstone/packages/cli
> pnpm build


> @webstone/cli@0.7.0 build /workspace/webstone/packages/cli
> pnpm clean && pnpm compile && pnpm copy-templates


> @webstone/cli@0.7.0 clean /workspace/webstone/packages/cli
> rm -rf ./dist


> @webstone/cli@0.7.0 compile /workspace/webstone/packages/cli
> tsc -p .

../../node_modules/.pnpm/gluegun@4.7.0/node_modules/gluegun/build/types/toolbox/prompt-enquirer-types.d.ts:70:34 - error TS2689: Cannot extend an interface 'NodeJS.EventEmitter'. Did you mean 'implements'?

70 declare class BasePrompt extends NodeJS.EventEmitter {
                                    ~~~~~~

../../node_modules/.pnpm/gluegun@4.7.0/node_modules/gluegun/build/types/toolbox/prompt-enquirer-types.d.ts:75:44 - error TS2689: Cannot extend an interface 'NodeJS.EventEmitter'. Did you mean 'implements'?

75 declare class Enquirer<T = object> extends NodeJS.EventEmitter {
  </pre>
</details>

A fix that works in my environment, but I'd need a maintainer's input on this, is to remove `extends NodeJS.EventEmitter` from [here](https://github.com/infinitered/gluegun/blob/master/src/toolbox/prompt-enquirer-types.ts#L99) and [here](https://github.com/infinitered/gluegun/blob/master/src/toolbox/prompt-enquirer-types.ts#L107).

FWIW, I found the Commander folks ended up with [the same fix for their project](https://github.com/tj/commander.js/pull/1146/files).

--- a/src/toolbox/prompt-enquirer-types.ts
+++ b/src/toolbox/prompt-enquirer-types.ts
@@ -1,7 +1,5 @@
 // imported from https://github.com/enquirer/enquirer/blob/8d626c206733420637660ac7c2098d7de45e8590/index.d.ts
 
-import { EventEmitter } from 'events'
-
 interface BasePromptOptions {
   name: string | (() => string)
   type: string | (() => string)
@@ -87,7 +85,7 @@ export type PromptOptions =
   | SnippetPromptOptions
   | SortPromptOptions
 
-declare class BasePrompt extends EventEmitter {
+declare class BasePrompt {
   constructor(options?: PromptOptions)
 
   render(): void
@@ -95,7 +93,7 @@ declare class BasePrompt extends EventEmitter {
   run(): Promise<any>
 }
 
-declare class Enquirer<T = object> extends EventEmitter {
+declare class Enquirer<T = object> {
   constructor(options?: object, answers?: T)
 
   /**

